### DISCUSSION TITLE
Add ability to sort peers by column (ex: up/down speed, country, %) in torrent menu

### DISCUSSION THREAD

**Discussion by itschasa** [jesec/flood#767] [Ideas]
Like how it is in the main table on the dashboard, allow the sorting of peers in individual torrent menu, just by all columns.
Country can be sorted by the country code.

![image](https://github.com/user-attachments/assets/1c1877b1-702c-425a-860a-5d3ea6bc1eef)

**Comment by trim21** [comment]
sounds like a good idea
